packages/node-sdk/sm/primitives/utils.ts

```diff
@@ -18,3 +18,13 @@ export function getScheduleBlockHeight<
   }
   return blockNumber;
 }
+
+// We cannot insert bigints into the database, or be serialized to JSON.
+export function clearBigInts<T>(value: T): T {
```

Comment:
remove this (also, you can put things like this in paima-utils)

Reply:
PAI-54